libtk205/include/libtk205.h

```diff
@@ -0,0 +1,38 @@
+#ifndef LIBTK205_H_
+#define LIBTK205_H_
+
+#include <vector>
+#include <memory>
+#include "ASHRAE205.h"
+#include "RS0001.h"
+#include "RS0002.h"
+#include "RS0003.h"
+#include "RS0004.h"
+#include "RS0005.h"
+#include "RS0006.h"
+#include "RS0007.h"
+#include "nlohmann/json.hpp"
+
+namespace libtk205_NS {
+
+    class A205_SDK 
```

Comment:
Do we need this class? I wonder if these can all be part of a single data file base class that all the "root" classes inherit from.

Reply:
It's not necessarily related. I was thinking that some of this functionality could be moved to the base class (e.g., read_binary_file).